src/test/scala-2.11/markets/participants/LiquiditySupplierSpec.scala

```diff
@@ -15,18 +15,18 @@ limitations under the License.
 */
 package markets.participants
 
-import akka.actor.ActorSystem
+import akka.actor.{ActorRef, ActorSystem}
 import akka.agent.Agent
 import akka.testkit.{TestProbe, TestActorRef, TestKit}
 
 import markets.orders.limit.LimitOrderLike
 import markets.tickers.Tick
-import markets.tradables.TestTradable
+import markets.tradables.{Tradable, TestTradable}
 import org.scalatest.{Matchers, GivenWhenThen, FeatureSpecLike}
 
+import scala.collection.mutable
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1703602240/issues/source?bid=2921361&fileBranchId=3046267#l27)